### PR TITLE
Add Azure ManifestWork support

### PR DESCRIPTION
**Pull request by jnpacker**
* Support Azure Cloud Provider credential via ManifestWork
* Include an AWS quickstart DRAFT
* Add Azure DiskSizeGB for 120GB
* Make sure we process the ProviderSecret when Configuration: False
  for Azure only
* Validated that Azure HypershiftDeployment was successful with ManifestWork

Signed-off-by: Joshua Packer <jpacker@redhat.com>

### CODE REVIEW

**Comment by jnpacker** [comment]
Issue reference: https://github.com/stolostron/backlog/issues/20477

**Comment by jnpacker** [comment]
ci/prow/sonar

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# [DRAFT] Quick start
```
Comment (ianzhang366) [inline docs/quickstart.md:1]: should we add an entry point on readme page?

Comment (jnpacker) [reply docs/quickstart.md:1]: Once we have the helm chart in MCE and we can skip the hypershift binary parts.

That's my feeling.

**Pull request merged**